easybuild/easyconfigs/m/MEME/MEME-5.1.1-foss-2019b-Perl-5.30.0-Python-3.7.4.eb

```
@@ -0,0 +1,39 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.1.1'
+versionsuffix = '-Perl-%(perlver)s-Python-%(pyver)s'
```

We have started to drop the superflous perl suffix, as we always stick to 1 version per toolchain anyway.
```suggestion
versionsuffix = '-Python-%(pyver)s'
```

---

Thanks, I was not aware of that. It is changed now.